Sortable.js

```diff
@@ -1002,7 +1002,7 @@
 					return el;
 				}
 			}
-			while (el = ('host' in el) ? el.host : el.parentNode)
+			while (el = ('host' in el && typeof el.host !== 'string') ? el.host : el.parentNode)
```

Comment:
Yes, but my error proves that at some place we have a string instead of nodeType.